README.md

```diff
@@ -5,8 +5,15 @@
 [![Code Climate](https://codeclimate.com/github/ocadotechnology/rapid-router/badges/gpa.svg)](https://codeclimate.com/github/ocadotechnology/rapid-router)
 [![Crowdin](https://d322cqt584bo4o.cloudfront.net/code-for-life/localized.svg)](https://crowdin.com/project/code-for-life)
 
-Rapid Router is a 'Code For Life' project, aimed at teaching children programming concepts through a vehicle routing
-game.
+## A  [Code for Life](https://www.codeforlife.education/) project
+* Rapid Router is a [Code for Life](https://www.codeforlife.education/) project, aimed at teaching children programming concepts through a vehicle routing
```

Comment:
Maybe "primary-school children" to contrast with the mention of aimmo further down.